SCons/Util.py

```diff
@@ -443,13 +508,19 @@ def flatten(obj, isinstance=isinstance, StringTypes=StringTypes,
             do_flatten(item, result)
     return result
 
-def flatten_sequence(sequence, isinstance=isinstance, StringTypes=StringTypes,
-                     SequenceTypes=SequenceTypes, do_flatten=do_flatten):
+
+def flatten_sequence(
+    sequence,
+    isinstance=isinstance,
+    StringTypes=StringTypes,
+    SequenceTypes=SequenceTypes,
+    do_flatten=do_flatten,
+):  # pylint: disable=redefined-outer-name,redefined-builtin
```

Comment:
can we annotate the return type here? `-> list` or sequence?

Reply:
yeah, where they were obvious I tried to put in the annotation (which, of course, is exactly when you're not adding much value). Guess I missed this one, since it starts by assigning an empty `[]` it's definitely going to be `list`.